src/reducers/index.ts

```diff
@@ -1,6 +1,8 @@
 import { combineReducers } from 'redux'
 import records from './records';
 
+
+
 export default combineReducers({
-  records
+  records: records
```

Comment:
The shorthand way of writing property values is fine and valid in TS. You can maintain the former and lose this change.